funcx_sdk/funcx/serialize/facade.py

```diff
@@ -1,21 +1,41 @@
+import atexit
+import multiprocessing as mp
+import subprocess
+import uuid
+import time
 from funcx.serialize.concretes import *
 from funcx.serialize.base import METHODS_MAP_DATA, METHODS_MAP_CODE
+from funcx.serialize.off_process_checker import OffProcessClient
 import logging
-
 logger = logging.getLogger(__name__)
 
 
 class FuncXSerializer(object):
     """ Wraps several serializers for one uniform interface
     """
 
-    def __init__(self):
+    def __init__(self, use_offprocess_checker=False):
```

Comment:
We probably want this always on? Or we should at least explicitly enable this in SDK client? This PR disables the off_process_checker.

Reply:
ah I think it makes sense. The serializer is used on the worker too. Then I think we need an option on `FuncXClient` to easily enable this?